### ISSUE TITLE
The databases need to include "Plasmids" too

### ISSUE THREAD

**Issue by tseemann** [DerrickWood/kraken#12]
Derrick,

I notice that often my pure bacterial samples still get 5% say of reads being unclassified. When I assemble these reads, they turn out to be bacterial plasmids.

The problem is that _some_ of the Bacteria folders have chromosomes and plasmids, but there are also many _separately submitted_ plasmids which are in a different Plasmids folder at NCBI:

```
ftp://ftp.ncbi.nih.gov/genomes/Plasmids/
```

It would be great to add support for this in the download tools, and in MiniKraken.

**Comment by DerrickWood** [comment]
Would the `plasmids.all.fna.tar.gz` file in that directory be sufficient for what you need? That would be easy enough to add to the download list.

**Comment by tseemann** [comment]
Yes, that would be sufficient.   This will help explain the unaccounted for 10-20% of reads in some samples where the plasmids have high copy number. Thanks!

**Comment by DerrickWood** [comment]
OK, I've added the plasmids as an option. I've got to monitor a few things over the next few months to make sure that file is consistently available before I can make it part of the standard installation, but at least people can more easily bring it in. I'll also likely add it to at least some version of the MiniKraken database in the near future as well.

**Comment by tseemann** [comment]
Thank you. 

The Plasmids folder has been there for many years, but not as long as Bacteria and Viruses, but NCBI do change things without notice.  The annoying part is that Bacteria/ does contain some plasmids, just not all.

**Comment by nickp60** [comment]
Hi, sorry to revive an old issue, but just to clarify, did the plasmids end up in either of the MiniKraken databases?

**Comment by jenniferlu717** [comment]
The plasmids in the plasmids folder did not end up in the minikraken databases but I'll work to include them in another version ASAP. Might take a couple of days though.

**Comment by tseemann** [comment]
@jenniferlu717 what ended up happening here?
